src/main/scala/wdlTools/cli/PrintAST.scala

```diff
@@ -0,0 +1,20 @@
+package wdlTools.cli
+
+import java.net.URI
+
+import wdlTools.syntax.ParseAll
+import wdlTools.util.{FetchURL, Util}
+
+import scala.language.reflectiveCalls
```

Comment:
Reflection is useful for what you want to do, that's cool, sometimes you need it. Could you try to keep the usage limited to the wdlTools.cli package?

Reply:
Yes - I believe it is required for using Scallop. It's the only downside to that library.